hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/HeapMemStoreLAB.java

```diff
@@ -206,6 +214,32 @@ private Chunk getOrMakeChunk() {
     }
   }
 
+  /** HBASE-14921
+   * Given a chunk ID return reference to the relevant chunk
+   * @return a chunk
+   */
+  public Chunk translateIdToChunk(int id) {
+    return chunkPool.translateIdToChunk(id);
```

Comment:
Here (and below) we don't assert that chunkPool != null. Seems to be better to give up on the chunkPool == null path altogether. Nobody wants it.